configs.signatures

```diff
@@ -381,7 +381,7 @@ declare -A configs_signatures=(
   ['7deb236ec68a512b9bdd18e6a51d76f7']='python.d/mysql.conf'
   ['7e5fc1644aa7a54f9dbb1bd102521b09']='health.d/memcached.conf'
   ['7f13631183fbdf79c21c8e5a171e9b34']='health.d/zfs.conf'
-  ['e48b89d4a97b96acf9a88970ab858c3b']='health.d/dbengine.conf'
+  ['f9ea512f96fc3971e1314d790e9a96ea']='health.d/dbengine.conf'
```

Comment:
We shouldnt change the signatures by hand.

Reply:
Is there a way to do it automatically? :D

Reply:
Well. If i remember it correctly this file is not used at all. We always rewrite stock config, we never rewrite user config. These signatures were used before stock/user configs separation.

See #6989. I updated `mysql.conf` there and i changed no singnatures.

Reply:
@knatsakis correct me i i am wrong